README.md

```diff
@@ -1,180 +1,108 @@
 # RFC Process
 
-This document describes the RFC process for the [npm
-CLI](https://github.com/npm/cli), and provides a way for the [npm CLI
-team](https://github.com/orgs/npm/teams/cli-team) and the wider community
-to have discussions about the features and direction of the package
-manager.
+This document describes the RFC process for the [npm CLI](https://github.com/npm/cli), and provides a way for the [npm CLI team](https://github.com/orgs/npm/teams/cli-team) and the wider community to have discussions about the features and direction of the package manager.
```

Comment:
It might be good to call out that the broader npm team may use this process to gather external feedback